README.md

```diff
@@ -30,6 +30,14 @@ Write your code here...
 
 ```ruby
 # code here
+
+
+def  offer_rose(person)
+  puts "Would you take this rose , person , in exchange for giving an old beggar woman shelter from the bitter cold?"
+end
+
+offer_rose("young prince")
+
 ```
```

Comment:
Just missing the string interpolation.

Answer:
  puts "Would you take this rose, `#{person}`, in exchange for giving an old beggar woman shelter from the bitter cold?"